app/src/main/java/com/duckduckgo/app/trackerdetection/blocklist/BlockList.kt

```diff
@@ -0,0 +1,139 @@
+        const val TREATMENT_URL = "treatmentUrl"
+        const val CONTROL_URL = "controlUrl"
+        const val NEXT_URL = "nextUrl"
+    }
+}
+
+@ContributesMultibinding(AppScope::class)
+class BlockListPixelsPlugin @Inject constructor(private val inventory: FeatureTogglesInventory) : MetricsPixelPlugin {
+
+    override suspend fun getMetrics(): List<MetricsPixel> {
+        val activeToggle = inventory.activeTdsFlag() ?: return emptyList()
+
+        return listOf(
+            MetricsPixel(
+                metric = "2xRefresh",
+                value = "1",
+                toggle = activeToggle,
+                conversionWindow = listOf(
+                    ConversionWindow(lowerWindow = 0, upperWindow = 0),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 1),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 2),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 3),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 4),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 5),
+                ),
+            ),
+            MetricsPixel(
+                metric = "3xRefresh",
+                value = "1",
+                toggle = activeToggle,
+                conversionWindow = listOf(
+                    ConversionWindow(lowerWindow = 0, upperWindow = 0),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 1),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 2),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 3),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 4),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 5),
+                ),
+            ),
+            MetricsPixel(
+                metric = "privacyToggleUsed",
+                value = "1",
+                toggle = activeToggle,
+                conversionWindow = listOf(
+                    ConversionWindow(lowerWindow = 0, upperWindow = 0),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 1),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 2),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 3),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 4),
+                    ConversionWindow(lowerWindow = 0, upperWindow = 5),
+                ),
+            ),
+        )
+    }
+}
+
+internal suspend fun BlockListPixelsPlugin.get2XRefresh(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "2xRefresh" }
+}
+
+suspend fun BlockListPixelsPlugin.get3XRefresh(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "3xRefresh" }
+}
+
+suspend fun BlockListPixelsPlugin.getPrivacyToggleUsed(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "privacyToggleUsed" }
+}
+
+suspend fun FeatureTogglesInventory.activeTdsFlag(): Toggle? {
```

Comment:
Only one experiment with prefix tds should be enabled at the same time, we are using this to always get the first one. If there are more then whoever is changing the remote config is doing it wrong. This was defined in https://app.asana.com/0/1125189844152671/1206056307411241/f but it's our way to cover for that case.